src/SQLClient/index.js

```diff
@@ -0,0 +1,17 @@
+const knex = require('knex');
+
+const SQLClient = (() => {
+  let instance;
```

Comment:
sholdn't  this `instance` be out of the `SQLClient` ? not sure how this scoping would work